Dockerfile

```diff
@@ -25,26 +25,24 @@ RUN uname -a
 COPY . .
 RUN go mod download &&\
     git submodule update --init --recursive &&\
-    bash `go env GOPATH`/pkg/mod/github.com/'linx!gnu'/grocksdb@v1.7.3/build.sh /shapleq/_thirdparty/rocksdb/install &&\
     make clean install DEPLOY_TARGET=release &&\
     cp -rf ~/.shapleq /shapleq/.shapleq
 
 FROM alpine:3.16
-RUN apk add --no-cache libstdc++ bash &&\
+RUN apk add --no-cache libstdc++ bash snappy &&\
     mkdir /shapleq
 
 COPY --from=builder /etc/passwd /etc/passwd
 COPY --from=builder /etc/group /etc/group
-COPY --from=builder /etc/group /etc/group
 COPY --from=builder /usr/local/bin/shapleq /shapleq/bin/shapleq
 COPY --from=builder /usr/local/bin/shapleq-client /shapleq/bin/shapleq-client
 COPY --from=builder /shapleq/start-shapleq.sh /shapleq/start-shapleq.sh
 COPY --from=builder /shapleq/.shapleq/config /shapleq/config
 
 WORKDIR /shapleq
-RUN chown -R shapleuser /shapleq
+RUN chown -R shapleuser /shapleq &&\
+    ls -al /shapleq/config/broker
 USER shapleuser:shapleuser
 EXPOSE 1101
-ENV ZK_ADDR=127.0.0.1
-ENV ZK_PORT=2181
-ENTRYPOINT ["bash", "-c", "./start-shapleq.sh $ZK_ADDR $ZK_PORT /shapleq/config/broker/config.yml"]
+ENV ZK_QUORUM="127.0.0.1:2181" HOME=/shapleq
+ENTRYPOINT ["bash", "-c", "./start-shapleq.sh $ZK_QUORUM /shapleq/config/broker/config.yml"]
```

Comment:
The ZK_QUORUM should be FQDN of zk service running on EKS. And hostname in broker/config.yml should be FQDN of broker service too.

Reply:
No. this Dockerfile is considered that using for general purpose. The ZK_QUORUM config value is changed by EKS setting, like deployment.yaml or statefulset.yaml, on a project-self

Reply:
You are right.